<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindTourist</class>
 <widget class="QDialog" name="FindTourist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find Tourist</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>290</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555;
font: 5pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
        </property>
        <property name="text">
         <string>EnterTourist Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtTouristName">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFind">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
        </property>
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
